test/helpers/index.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
+  comparePoints: require('./comparePoints'),
   comparePolygons: require('./comparePolygons'),
+  comparePolygonsAsPoints: require('./comparePolygonsAsPoints'),
```

Comment:
I do not see this added as part of this PR ? or am I missing something

Reply:
Just updating the index. The helpler function has been there for a long time.